Shared/src/Datahub.Shared/Entities/AppServiceConfiguration.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace Datahub.Shared.Entities
+{
+    public class AppServiceConfiguration
+    {
+        public string AppServiceFramework { get; set; } = string.Empty;
+        public string AppServiceGitRepo { get; set; } = string.Empty;
+        public string AppServiceComposePath { get; set; } = string.Empty;
+        public string AppServiceId { get; set; } = string.Empty;
+        public string AppServiceHostName { get; set; } = string.Empty;
+        public string AppServiceRg { get; set; } = string.Empty;
```

Comment:
Just a thought that you can probably omit the `AppService` bit from the nested variables 😄 

Since it'll be read as something like `workspaceAppServiceConfiguration.GitRepo`

Reply:
and then you can expand `Rg` into `ResourceGroupName` 😉